Terminal.Gui/Views/Menu.cs

```diff
@@ -2099,5 +2099,30 @@ public override bool OnEnter (View view)
 
 			return base.OnEnter (view);
 		}
+
+		/// <summary>
+		/// Gets the superview location offset relative to the <see cref="ConsoleDriver"/> size 
+		/// which is measured by the <see cref="ConsoleDriver.Cols"/> and <see cref="ConsoleDriver.Rows"/>.
+		/// </summary>
+		/// <returns>The location offset.</returns>
+		internal Point GetDriverLocationOffset ()
```

Comment:
This code is very hard to understand.

You reference `Driver.Cols/Rows` but never use the height or width of `superViewFrame`.

I think this is what happens (just for X):

If SuperView != null then
   return SuperView.Frame.X - SuperView.Frame.X - (SuperViewFrame.X != SuperView.Frame.X ? 1 : 0)
else
   return 0 - Application.Current.Frame.X - (0 != Application.Current.Frame.X ? : 1 : 0)

Reply:
I think you want 

```cs
if (SuperView != null) {
  return new Point (SuperView.Frame.X - 1, SuperView.Frame.Y - 1);
} else {
  return new Point (Application.Current.Frame.X - 1, Application.Current.Frame.Y - 1);
}
```

Isn't that the same (and more clear)?

(Although I can't figure out what the -1 is for).

Reply:
It's the bounds offset. I'll change that.